wpeverest/user-registration
# Enhancement - User registration gutenberg blocks

### All Submissions:


### Changes proposed in this Pull Request:
This pull request breaks the old user registration block into three separate blocks: registration form, login form, and my account. It also adds two new blocks: edit profile and edit password.
### How to test the changes in this Pull Request:
1. Switch into the branch and run : npm install > npm run start or npm run build.
2. Edit or create a page
3. Click on toggle block inserter
4. Lookup for user registration blocks and drag and verify its functionality both in frontend and backend editor.

### Types of changes:


### Other information:


### Changelog entry

> Enhancement - User registration gutenberg blocks.
